src/inc/sstring.h

```diff
@@ -1009,9 +1012,11 @@ typedef InlineSString<32>  SmallStackSString;
 #ifdef _DEBUG
 // This is a smaller version for debug builds to exercise the buffer allocation path
 typedef InlineSString<32> PathString;
+typedef InlineSString<2 * 32> LongPathString;
 #else
 // Set it to the current MAX_PATH
 typedef InlineSString<260> PathString;
+typedef InlineSString<2 * 260> LongPathString;
```

Comment:
LongPathString is exclusively used by the wrappers in coreclr\src\utilcode\longfilepathwrappers.cpp. Where when we try to normalize the path there is a good chance that the length could exceed MAX_PATH.